R/woods_plot.R

```diff
@@ -1,27 +1,34 @@
 #' length and optionally a variable based on which peptides or precursors should be coloured.
-#' @param fold_change Column in the data frame containing log2 fold changes.
-#' @param start_position Column in the data frame containing the start positions for each peptide or precursor.
-#' @param end_position Column in the data frame containing the end positions for each peptide or precursor.
-#' @param protein_length Column in the data frame containing the length of the protein.
-#' @param coverage Optional, column in the data frame containing coverage in percent. Will appear in the title of the barcode if provided.
-#' @param protein_id Optional argument, column in the data frame containing protein identifiers. Required if only one protein
-#' should be plotted and the data frame contains only information for this protein.
-#' @param facet Optional argument, column in the data frame containing information by which data should be faceted. This can be
-#' protein identifiers.
-#' @param colouring Optional argument, column in the data frame containing information by which peptide or precursors should
+#' @param fold_change a column in the data frame containing log2 fold changes.
+#' @param start_position a column in the data frame containing the start positions for each peptide or precursor.
+#' @param end_position a column in the data frame containing the end positions for each peptide or precursor.
+#' @param protein_length a column in the data frame containing the length of the protein.
+#' @param coverage optional, column in the data frame containing coverage in percent. Will appear in the title of the barcode if provided.
+#' @param protein_id a column in the data frame containing protein identifiers.
+#' @param targets a character vector that specifies the identifiers of the proteins (depending on \code{protein_id})
+#' that should be plotted. This can also be \code{"all"} if plots for all curve fits should be created. Default is \code{"all"}.
+#' @param facet a logical indicating if plots should be summarised into facets of 20 plots. This is recommended for many plots.
+#' Default is \code{facet = TRUE}.
+#' @param colouring optional, column in the data frame containing information by which peptide or precursors should
 #' be coloured.
-#' @param fold_change_cutoff Optional argument specifying the log2 fold change cutoff used for assessing whether changes are significant. The default value is 2.
+#' @param fold_change_cutoff optional, numeric argument specifying the log2 fold change cutoff used in the plot. The default value is 2.
+#' @param highlight optional logical column containing logicals, specifying whether specific peptides or precursors should be highlighted with an asterisk.
+#' @param export a logical indicating if plots should be exported as PDF. The output directory will be the current working directory. The
+#' name of the file can be chosen using the \code{export_name} argument. Default is \code{export = TRUE}.
```

Comment:
I would set the default to FALSE as we do for the plot_drc_4p function

Reply:
actually just saw that you set this to FALSE anyway, so just change this here in the description